vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -509,3 +510,18 @@ type Runtime struct {
 	Path string   `json:"path"`
 	Args []string `json:"runtimeArgs,omitempty"`
 }
+
+type ContainerStats struct {
+	Name             string
+	CPUPercentage    float64
+	Memory           float64
+	MemoryLimit      float64
+	MemoryPercentage float64
+	NetworkRx        float64
+	NetworkTx        float64
+	BlockRead        float64
+	BlockWrite       float64
+	PidsCurrent      uint64
+	Mu               sync.Mutex
+	Err              error
```

Comment:
@cpuguy83 if we keep the `types.ContainerStats` in the `container` package then we will have import cycle:
```
import cycle not allowed
package github.com/docker/docker/cmd/docker
        imports github.com/docker/docker/cli/cobraadaptor
        imports github.com/docker/docker/api/client/container
        imports github.com/docker/docker/api/client/formatter
        imports github.com/docker/docker/api/client/container
```
Anyway, to avoid this cycle, that structure should moved to another package. engine-api's type seems like a good place.